native/NativeSampleApps/RestExplorer/res/values/bootconfig.xml

```diff
@@ -5,6 +5,7 @@
     <string name="oauthRedirectURI">testsfdc:///mobilesdk/detect/oauth/done</string>
     <string-array name="oauthScopes">
         <item>api</item>
+        <item>web</item>
```

Comment:
Need `web` scope for frontdoor and to act as an IDP.

Reply:
Are we going to put a check in the code to make sure the web scope is available or are we just let it fail in the web view?

Reply:
In the next PR, I will add more error handling. I plan on adding the error callback within `WebViewClient`, so that if an error callback occurs, I can propagate the error back.